Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoEmptyBlockRule.swift

```diff
@@ -0,0 +1,114 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct NoEmptyBlockRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "no_empty_block",
+        name: "No Empty Block",
+        description: "Code blocks should not be empty; they should at least contain a comment",
```

Comment:
I'm not too happy with this message yet. Twice the "should" bugs me. @mildm8nnered, could you help out?

Reply:
So looking through the other rule descriptions, they are mostly single clauses, and negative ("should not"), but sometimes positive. How about something like:

"Code blocks should contain at least one statement or comment"

Or, using "avoid", which occurs in some existing rule descriptions:

"Empty blocks with no statements or comments should be avoided"

Reply:
I like the first suggestion.